src/persistence/Elsa.Persistence.DocumentDb/Services/DatabaseRegister.cs

```diff
@@ -0,0 +1,38 @@
+using Elsa.Models;
+using Elsa.Services.Models;
+using System;
+
+namespace Elsa.Persistence.DocumentDb.Services
+{
+    public static class DatabaseRegister
+    {
+        public static void RegisterMapsAndSerializers()
+        {
+            // In unit tests, the method is called several times, which throws an exception because the entity is already registered
+            // If an error is thrown, the remaining registrations are no longer processed
+            var firstPass = Map();
+
+            if (firstPass == false)
+                return;
+
+            RegisterSerializers();
+        }
+
+        private static bool Map()
+        {
+            try
+            {
+            }
+            catch (Exception)
+            {
+                return false;
+            }
+
+            return true;
+        }
+
+        private static void RegisterSerializers()
+        {
+        }
+    }
+}
```

Comment:
Do we need to keep this class? It doesn't look like it's doing anything special.

Reply:
no, sorry, I forgot it :D